pkg/api/routes.go

```diff
@@ -60,6 +60,12 @@ func (h *Handler) GetRoutes() []Route {
 			h.CreateImage,
 		},
 		{
+			"DeleteImage",
+			"POST",
```

Comment:
@DanKerns The rook volume will make use of the API to create an image. I dont think creating an image should be done by the rook-operator and with TPR. I see the rook-operator more of a deployer, maintainer, configurer and monitor of the rook-cluster. Creating an image seems to me more of a user-consumption request. Besides kubernetes already has a resource type for block images. It is called PersistentVolume :)